emu/decode.h

```diff
@@ -301,6 +309,9 @@ __no_instrument DECODER_RET glue(DECODER_NAME, OP_SIZE)(DECODER_ARGS) {
                 case 0xc5: TRACEI("pextrw xmm, modrm_val, imm8");
                            READMODRM_NOMEM; READIMM8; V_OP_IMM(extract_w, xmm_modrm_val, modrm_reg,128); break;
 
+                case 0xd4: TRACEI("paddq xmm:modrm, xmm");
+                           READMODRM; V_OP(add, xmm_modrm_val, xmm_modrm_reg, 64); break;
```

Comment:
Should be 128 instead of 64, the size param specifies how big the memory access should be. Ditto for psrlq and psllq

Reply:
Gotcha, so the function definitions should change to addq128 or something along those lines too?

Reply:
Yeah, I've been specifying the component size as part of the op name. For example, shuffle_d, or add_q